colcon_ros/task/ros.py

```diff
@@ -75,6 +75,32 @@ def add_arguments(self, *, parser):  # noqa: D102
                 if args.cmake_args is None:
                     args.cmake_args = []
                 args.cmake_args += args.ament_cmake_args
+            # extend CMAKE_PREFIX_PATH with AMENT_PREFIX_PATH
+            ament_prefix_path = os.environ.get('AMENT_PREFIX_PATH')
+            if ament_prefix_path:
+                if args.cmake_args is None:
+                    args.cmake_args = []
+                # check if the CMAKE_PREFIX_PATH is explicitly set
+                prefix = '-DCMAKE_PREFIX_PATH='
+                for i, value in reversed(list(enumerate(args.cmake_args))):
```

Comment:
Reversed because the last `-DCMAKE_PREFIX_PATH=` if there are multiple is the one that takes effect?

Reply:
Correct: https://github.com/colcon/colcon-ros/pull/21/files#diff-9b50740cc6d002781d0475ec643e9034R88